docs/.doctor-rst.yaml

```diff
@@ -1,7 +1,7 @@
 rules:
     american_english: ~
     avoid_repetetive_words: ~
-    blank_line_after_directive: ~
+    # blank_line_after_directive: ~
```

Comment:
Why do you need to deactivate this rule?

Reply:
See: https://github.com/phpbench/phpbench/pull/713/files/8a0e5cc81653ab7058933223ec08481b8ef53e89..879b034a9fb9ddf80d84c09ea82908ef0decac55

(I reverted `before_line_after` as that didn't cause issues)